web/src/stories/icon.stories.mdx

```diff
@@ -53,6 +53,32 @@ export const iconWithTooltipView=  {
   }
 };
 
+export const svg = `
```

Comment:
The validation might be too aggressive. This is valid input currently being filtered out as seen in storybook